src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -401,6 +401,20 @@ protected function createChildContext(array $parentContext, $attribute/*, string
     private function getCacheKey($format, array $context)
     {
         unset($context['cache_key']); // avoid artificially different keys
+
+        if (interface_exists(\Throwable::class)) {
```

Comment:
that's not needed. You can add 2 `catch` block, and catching a non-existent class is not an issue (that's an `instanceof` check).

Reply:
Nice, I wasn't aware of that!